content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/_cq_design_dialog/.content.xml

```diff
@@ -98,15 +98,17 @@
                                         text="Display caption as pop-up"
                                         uncheckedValue="false"
                                         value="{Boolean}true"/>
-                                    <disableUuidTracking
+                                    <resizeWidth
                                         jcr:primaryType="nt:unstructured"
-                                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                        fieldDescription="Check to disable tracking of asset's UUID."
-                                        name="./uuidDisabled"
-                                        text="Disable UUID tracking"
-                                        checked="{Boolean}true"
-                                        uncheckedValue="false"
-                                        value="{Boolean}true"/>
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                        fieldDescription="Define a resize width (in pixels). Its value will be used for resizing the width of the base images that are DAM assets. The aspect ratio of the images will be preserved. If the value is bigger than the natural width of an image, it will have no effect. Note that this setting no effect on SVG images."
```

Comment:
There seems to be a typo:
Note that this setting no effect on SVG images. -> "Note that this setting has no effect on SVG images."